### PR TITLE
chore(license): add copyright

### PR DESCRIPTION
**Pull request by priteshbandi**
update License: Added copyright

References:
1. ORAS - https://github.com/oras-project/oras-go/blob/main/LICENSE#L189
2. Argo - https://github.com/adobe/argo/blob/master/LICENSE#L190
3. fluentd - https://github.com/fluent/fluentd/blob/master/LICENSE#L190

Signed-off-by: Pritesh Bandi <pritesb@amazon.com>

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020 The Notation Authors
```
Comment (shizhMSFT) [inline LICENSE:189]: How do we determine the year? Should it be `2022` (the 1st year of this repo) or `2020` (the 1st year of the notary v2 work)? or even `2019` (the year we kick off the project)?

Comment (priteshbandi) [reply LICENSE:189]: For determining year I looked at the date of [first commit](https://github.com/notaryproject/notation/tree/560d4f8f56546c51ee6c3e5a7241f1961cd8bd78) in Notation and used that for all 3 Notation repos.
PRs for all repo:
https://github.com/notaryproject/notation/pull/536
https://github.com/notaryproject/notation-go/pull/267
https://github.com/notaryproject/notation-core-go/pull/116

Comment (toddysm) [reply LICENSE:189]: Shouldn't this actually be?
> Copyright 2020-2022 The Notation Authors

This spans all the years development was in progress.

**Comment by priteshbandi** [comment]
Closing PR as per community discussion; (other CNCF graduated projects still have `[yyyy] [name of copyright owner]` in there license.

**Pull request closed without merging**